docs/_getting-started/index.md

```diff
@@ -4,197 +4,311 @@ permalink: docs/getting-started
-standalone pre-built binary, running it as a docker container or building it yourself.
+To use Geth, you need to install it first. You can install Geth in a variety of ways that you can find in the “[Install and Build](install-and-build/installing-geth)” section. 
+These include installing it via your favorite package manager, downloading a standalone pre-built binary, running it as a docker container, or building it yourself.
 
-For this guide, we assume you have Geth installed and are ready to find out how to use it.
-The guide shows you how to create accounts, sync to a network, and then send transactions
-between accounts.
+For this guide, we assume you have Geth installed and are ready to find out how to use it. 
+The guide shows you how to create accounts, sync to a network, and then send transactions between accounts.
+This guide uses [Clef](clef/tutorial), which is our preferred tool for signing transactions with Geth, and will replace Geth’s account management.
 
-This guide uses [Clef](clef/tutorial), which is our preferred tool for signing transactions with Geth,
-and will replace Geth's account management.
+### Two Important terms in geth:
 
-## Initialize Clef
+- Network
+- Sync modes
 
-First, initialize Clef with a random master seed, which is also encrypted with the password you define.
-The password must be at least 10 characters.
+### Networks
+You can connect a Geth node to several different networks using the network name as an argument. These include the main Ethereum network, [a private network](getting-started/private-net)you create, and three test networks that use different consensus algorithms:
 
-```shell
-clef init
-```
+-   Ropsten: Proof-of-work test network
+-   Rinkeby: Proof-of-authority test network
+-   Görli: Proof-of-authority test network
 
-![Clef init command](../../static/images/clef-init.gif)
+For this guide, you will use the Görli network. The default port is 8545, so you need to enable at least outgoing access from your node to that port.
+### Sync modes
+You can start Geth in one of three different sync modes using the `--syncmode "<mode>"` argument that determines what sort of node it is in the network.
+These are:
 
-## Create accounts
+- Full: Downloads all blocks (including headers, transactions, and receipts) and generates the state of the blockchain incrementally by executing every block.
+- Snap (Default): Same functionality as fast, but with a faster algorithm.
+- Light: Downloads all block headers, block data, and verifies some randomly.
 
-Create two accounts with the `clef newaccount` command,
-set a password for each of them, and note the public address for each.
+For this tutorial, you will use a `light` sync:
 
-![Create new account command](../../static/images/clef-account-new.gif)
+### Prerequisites:
 
-Clef outputs a debug message about `Failed to reload keystore contents`, we fix that in a later step.
+- Curl experience 
+- Command line
+- Basic Blockchain Knowledge
```

Comment:
It's not a bad idea to list the prerequisites for this guide, but you absolutely need to be more specific here. What we are expecting is:

- Experience using the command line
- Basic knowledge about Ethereum and testnets
- Basic knowledge about HTTP and JavaScript